onnxruntime/core/optimizer/selectors_actions/selector_action_transformer.h

```diff
@@ -86,7 +95,7 @@ class SelectorActionRegistry {
 #else  // !defined(ORT_MINIMAL_BUILD)
 
   // register an action
-  void RegisterAction(const std::string& name, std::unique_ptr<Action> action);
+  void RegisterAction(const std::string& name, std::unique_ptr<Action> action, const std::string& domain = kOnnxDomain);
```

Comment:
do we still need a domain parameter here?

Reply:
Removed the domain parameter.